reefperf/utils.py

```diff
@@ -7,3 +11,21 @@ def __call__(cls, *args, **kw):
         if cls.instance is None:
             cls.instance = super(Singleton, cls).__call__(*args, **kw)
         return cls.instance
+
+
+def get_node_resources(deploy_script_output):
+    node_resources = {}
+    capture_line = False
+    for line in deploy_script_output.decode('utf-8').split('\n'):
+        line = line.strip()
```

Comment:
if someone will want to pass a randomly generated password with a space at the end, this won't work. Maybe use `rstrip('\n')` instead?